foxy/distribution.yaml

```diff
@@ -669,7 +669,7 @@ repositories:
       tags:
         release: release/foxy/{package}/{version}
       url: https://github.com/canlab-co/CLPE_G_NVP2650D_SDK-ros2-release.git
-      version: 0.1.0-5
+      version: 0.1.0-1
```

Comment:
@rorosi mind to explain the reverse bump?

Reply:
I accidentally deleted the repository. :(